doc/tutorials/setup_openstack_libvirt.rst

```diff
@@ -59,8 +55,9 @@ Set up and launch LXC/LXD
      backend).
    - Answer ``dir`` (and not ``zfs``) when prompted for the name of the storage backend to use.
    - Set up ``lxdbr0`` as instructed.
-   - Before launching a container, make sure ``lxc-checkconfig | grep missing`` does not show any missing
-     kernel features.
+   - Before launching a container, install lxc-utils by ``apt-get install lxc-utils``,
+     make sure ``lxc-checkconfig | grep missing`` does not show any missing kernel features
+     except CONFIG_NF_NAT_IPV4/CONFIG_NF_NAT_IPV6 which are changed from the kernel.
```

Comment:
```suggestion
     except ``CONFIG_NF_NAT_IPV4`` and ``CONFIG_NF_NAT_IPV6`` which have been renamed in very recent kernels.
```

Reply:
I made this change and merged this version.  We'll explain the hugepages settings after the v2.3 release.